ledger/light/proc/hot_data.go

```diff
@@ -32,17 +34,23 @@ import (
 	"github.com/insolar/insolar/ledger/object"
 )
 
+const (
+	pendingNotifyThreshold = 2
```

Comment:
why `2`? it would be nice to have some explanation for any magic constant in code.